Tour Manager backend without the calendar

RBasentsyan opened the pull request:
Some minor improvements are needed. For example, during the GetAllTours call, when getting the participants, it retrieves only the first participant. This is not the case when getting a single tour by Id. 
Due to time constraints, I was not able to fix that bug.

Commit:
Tour Manager backend without the calendar

arttonoyan commented on src/TourManager/Travely.TourManager.API/Startup.cs:
```diff
@@ -0,0 +1,83 @@
+
+            services.AddMvc().AddNewtonsoftJson();
+
+            services.AddScoped<ITourTypeService, TourTypeService>();
+            services.AddScoped<ITourService, TourService>();
+            services.AddScoped<ITourStatusService, TourStatusService>();
+            services.AddScoped<ILanguageService, LanguageService>();
+            services.AddScoped<IGenderService, GenderService>();
```
Please, create `ServiceCollectionExtensions` in the services layer and move injection logic there

arttonoyan commented on src/TourManager/Travely.TourManager.DAL/Attachment.cs:
```diff
@@ -0,0 +1,22 @@
+    public class Attachment
+    {
+        [Key]
+        [Column("Id")]
```
Move configurations logic to `IEntityTypeConfiguration` or to `OnModelCreating`.

Commit:
Created ServiceCollectionExtensions class for the injection logic
Moved the configuration logic to OnModelCreating
Added a ToDo.txt file which contains the bugs that need to be fixed and the features that need to be added
Moved all the services in Travely.TourManager.BLL to a new folder called "Services"
